README.md

```diff
@@ -0,0 +1,8 @@
+# Team Red GameEngine
+## How to build
+1. You'll need to download the latest version of CMake https://cmake.org/download/
```

Comment:
Maybe specify this because I got stuck on it:
You'll need to download the latest version of CMake https://cmake.org/download/ and add it to your PATH environment variable (This is easiest if you download the Binary Installer version)

Reply:
Added more detail